java/code/src/com/suse/manager/webui/websocket/RemoteMinionCommands.java

```diff
@@ -265,13 +269,19 @@ else if (previewedMinions.isEmpty()) {
 
                 res.forEach((minionId, future) -> future.whenComplete((cmdResult, err) -> {
                     if (cmdResult != null) {
-                        sendMessage(session,
-                                cmdResult.fold(
-                                        error -> new ActionErrorEventDto(minionId,
-                                                "ERR_CMD_SALT_ERROR",
-                                                parseSaltError(error)),
-                                        result -> new MinionCommandResultEventDto(minionId,
-                                                result)));
+                        AbstractSaltEventDto event = cmdResult.fold(
+                                error -> {
+                                    String errMsg = parseSaltError(error);
+                                    LOG.info("Received Salt error for minion " + minionId + ": " + errMsg);
+                                    return new ActionErrorEventDto(minionId,
+                                        "ERR_CMD_SALT_ERROR", errMsg);
+                                },
+                                result -> {
+                                    LOG.info("User '" + webSession.getUser().getLogin() +
+                                            "' received command result from minion " + minionId + ": " + result);
```

Comment:
I don't think we need to log the actual contents of the result/error. These can get arbitrarily big, and probably not necessary for audit purposes anyway. Only logging which user ran what command on what machines, and whether it was successful or not would be sufficient IMO.

Reply:
Or maybe we can print the contents on a separate line with log level `DEBUG`.

Reply:
Logged output separately at `DEBUG` level.